File: ABC/<100/090/Julia/D.jl
parInts(str=readline()) = parse.(Int, str|>split)
function main()
  N, K = parInts()
  answer = 0
  for b ∈ K+1:N
    answer += (N÷b)*(b-K) + max((N%b - K + 1), 0)
    answer -= (K==0 ? 1 : 0)
  end
  answer |> println
end
main()

File: ABC/<100/090/C++/D.cpp
#include<iostream>
using namespace std;
typedef int64_t ll;
#define rep(i, s, t) for(ll i = (s); i < (t); ++i)
#define ll0 (ll)0
#define ll1 (ll)1

int main(){
  ll N, K; cin >> N >> K;

  ll answer = ll0;
  rep(b, K+1, N+1){
    answer += (N/b) * (b-K) + max(N%b - K + ll1,  ll0);
    answer -= (K == ll0 ? ll1 : ll0);
  }
  cout << answer << endl;
  return 0;
}